src/Uno.UI.RemoteControl.VS/EntryPoint.cs

```diff
@@ -10,18 +10,19 @@
 using System.Reflection;
 using System.Threading;
 using System.Threading.Tasks;
-using System.Windows.Forms;
 using EnvDTE;
 using EnvDTE80;
 using Microsoft.Build.Evaluation;
 using Microsoft.Internal.VisualStudio.Shell;
+using Microsoft.VisualStudio.Imaging;
```

Comment:
Is this namespace really needed ? Sounds weird

Reply:
We need the namespace for the [KnownMonikers ](https://learn.microsoft.com/en-us/dotnet/api/microsoft.visualstudio.imaging.knownmonikers?view=visualstudiosdk-2022)